lib/membrane-element-rtp-aac/depayloader.ex

```diff
@@ -5,8 +5,8 @@ defmodule Membrane.RTP.AAC.Depayloader do
   alias Membrane.Buffer
   alias Membrane.{AAC, RTP}
 
-  def_input_pad :input, demand_unit: :buffers, caps: RTP
-  def_output_pad :output, caps: {AAC, encapsulation: :none}
+  def_input_pad :input, demand_unit: :buffers, accepted_format: %RTP{}
```

Comment:
`demand_unit: :buffers` - does it change anything? If it is a default, remove it

Reply:
done